cmd/bosun/expr/funcs.go

```diff
@@ -355,7 +367,7 @@ func parseGraphiteResponse(req *graphite.Request, s *graphite.Response, formatTa
 		// build tag set
 		nodes := strings.Split(res.Target, ".")
 		if len(nodes) < len(formatTags) {
-			return nil, fmt.Errorf(`returned target "%s" does not match format "%s"`, res.Target, formatTags)
+			return nil, graphite.NewParseError(fmt.Sprintf("returned target '%s' does not match format '%s'", res.Target, strings.Join(formatTags, ",")), nil, req.Url)
```

Comment:
this tracks the request url as a separate attribute. if we were to put everything together than the error message itself would be too long, we wouldn't have the option of [not] showing the url, and we would have to decide up front how the url should be shown (in dashboard and log we do it in a new line, but i don't think we should formatting such as newlines into the error message)